chapters/infrastructure/digital-rewards/reading/README.md

```diff
@@ -3,10 +3,38 @@
 Owner: Răzvan Nițu
 Reviewer: Mihai Băruța
 
-definitions, types
+Some teaching paradigms involve the use of a reward system to incentivize learners to study.
+To enable the use of such teaching techniques, we offer an infrastructure for the use of cryptocurrency rewards that may be used to stimulate both learning and contributions to the educational content.
+We do not discuss the efectiveness of reward-based learning techniques, we simply provide the tools to implement such a system if an educator chooses to use it.
 
-goals of using digital rewards
+Rewards may be used at different steps in the learning process:
 
-infrastructure for digital rewards
+- when a learner correctly completes a designated teaching activity: a drill or set of drills, an assignment, an exam etc.
+- when a learner contributes content to the existing repo
+- when a learner helps his peers (by publicly answering forum questions or helping them during practical sessions)
 
-cryptocurrency
+Choosing the items for which rewards are offered is a subjective decision depending on personal preference and cultural background.
+However, from a usability perspective, there are two categories of rewards:
+
+- those that can be awarded automatically (e.g. a learner solves a multiple choice question)
+- those that are awarded manually (e.g. a learner has answered a verbal question during a lecture)
+
+To account for these situations, we have developed an infrastructure that uses an educational cryptocurrency that may be awarded automatically or manually.
```

Comment:
Should we add some links to the infrastructure repositories?

- https://github.com/Jokeswar/smileycoin-api
- https://github.com/Jokeswar/smileycoin-node